### PR TITLE
Update override classes in examples/typography.

### PR DESCRIPTION
**Pull request by hannalaakso**
This was missed out from #368 as the typography classes changed.

Also update docs on override classes.

### CODE REVIEW

**Comment by dashouse** [comment]
I think these might be using the previous values (as they weren't changed after the spacing scale changes) so anything above the 45px unit will need to go up one in the scale. That said, this is just for the review page so I'm not too fussed :)

**Comment by hannalaakso** [comment]
That's a valid point. I've just pushed an amend if you don't mind reviewing that.

**Comment by dashouse** [comment]
@hannalaakso Perfect, looks great

File: docs/coding-standards/css.md [hunk 1]
```diff
@@ -72,7 +72,8 @@ Classes that override all other layers - for example to override the spacing or
 font size / line height on an element.
 
 - forms part of the @govuk-frontend/global package.
-- uses the prefix `govuk-!-` for its classes.
+- uses the prefix `govuk-!-` for its classes, eg. `govuk-!-mt-r8`; `r` indicates the class is responsive and `8` is the scale point; these are set in settings/spacing
```
Comment (36degrees) [inline docs/coding-standards/css.md]: 👌

**Pull request merged**